gm4_trash_cans/data/gm4_trash_cans/functions/delete_items.mcfunction

```diff
@@ -0,0 +1,7 @@
+# @s = trash_can that has items in it and has not been disabled
+# run from check_rate
+
+# Decrease items in trash can by the rate of deletion
+execute store result score @s gm4_trash_amount run data get block ~ ~ ~ Items.[0].Count
+scoreboard players operation @s gm4_trash_amount -= @s gm4_trash_rate
+execute store result block ~ ~ ~ Items.[0].Count byte 1 run scoreboard players get @s gm4_trash_amount
```

Comment:
Why does the trash can only affect items in slot 0?

Reply:
Its looking at the first item in the array not the first slot in the inventory.